.github/workflows/ci.yaml

```diff
@@ -458,48 +458,9 @@ jobs:
       MIRRORD_AGENT_RUST_LOG: "warn,mirrord=debug"
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rust-lang/setup-rust-toolchain@v1
-        with:
-          components: rustfmt
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 14
-      - run: npm install express
-      - uses: actions/setup-python@v3
-      - run: pip3 install flask
-      - run: pip3 install fastapi
-      - run: pip3 install uvicorn[standard]
-      - run: sudo apt-get update && sudo apt-get install -y curl
-      - uses: actions/setup-go@v3
-        with:
-          go-version: "1.18.0"
-      - run: |
-          go version
-      - run: | # Build Go test apps.
-          ./scripts/build_go_apps.sh 18
-      - uses: actions/setup-go@v3
-        with:
-          go-version: "1.19.0"
-      - run: |
-          go version
-      - run: | # Build Go test apps.
-          ./scripts/build_go_apps.sh 19
-      - uses: actions/setup-go@v3
-        with:
-          go-version: "1.20.0-rc.3"
       - run: |
-          go version
-      - run: | # Build Go test apps.
-          ./scripts/build_go_apps.sh 20
-      - run: |
-          cd tests/rust-e2e-fileops
-          cargo build
-      - run: |
-          cd tests/rust-unix-socket-client
-          cargo build
-      - run: |
-          cd tests/rust-bypassed-unix-socket
-          cargo build
+          chmod +x ./scripts/setup_e2e.sh
```

Comment:
I think you can do it with `git update-index --chmod=+x` beforehand on the file so it will be saved as an executable